annotations/templates/annotations/concept_add.html

```diff
@@ -77,7 +77,7 @@
     </ul>
     {% endif %}
 
-    <a class="btn btn-md btn-danger" href="{{ next_page }}">Take me back!</a>
+    <a class="btn btn-md btn-danger" href="{% url "concepts" %}">Take me back!</a>
```

Comment:
I believe this does not pay attention to the page the user was on? If I come to this page from the 4th page of the concepts, this will not take me back to the 4th page, will it? It should though.